### DISCUSSION TITLE
the builds in Spoke wont appear in Hubs

### DISCUSSION THREAD

**Discussion by vanessa23458** [mozilla/hubs#4215] [Spoke Questions] [answered]
My name is Vanessa and im new at Hubs and i tried to create a scene but when i build stuff on Spoke they dont appear in Hub. What can i do about it?

**Comment by j-conrad** [comment, marked as answer]
Make sure that you are using the '**_Publish to Hubs_**' button and going through the dialogs that follow. Once you see the progress bar finish uploading, you'll get a button to view your scene in Hubs.
Unless you're saying that part of the scene or some object doesn't show, which would be a different issue.

**Reply by Softvision-GeluHaiduc** [reply]
Marking this as answered, unmark the answer if you need any more information. Thank you!

**Comment by vanessa23458** [comment]
Hello,

I managed to create my room on another computer but now i was wondering if there is a way to open a room for a week without closing the computer. Thank you so much for your help.

________________________________
